server/src/main/java/io/crate/planner/node/management/ExplainPlan.java

```diff
@@ -207,8 +207,12 @@ private CompletableFuture<?> executeTopLevelPlan(LogicalPlan plan,
         timer.start();
         BaseResultReceiver resultReceiver = new BaseResultReceiver();
         RowConsumer noopRowConsumer = new RowConsumerToResultReceiver(resultReceiver, 0, t -> {});
-        NodeOperationTree operationTree = LogicalPlanner.getNodeOperationTree(
-            plan, executor, plannerContext, params, subQueryResults);
+        NodeOperationTree operationTree = null;
+        try {
+            operationTree = LogicalPlanner.getNodeOperationTree(plan, executor, plannerContext, params, subQueryResults);
+        } catch (Throwable e) {
+            consumer.accept(null, e);
+        }
```

Comment:
Similar to https://github.com/crate/crate/blob/c52cedb35132d477bbbb01d019975ecea5180da5/server/src/main/java/io/crate/planner/operators/LogicalPlanner.java#L598-L603

Reply:
The first commit is enough for the reported scenario but just for the completeness...